### ISSUE TITLE
Add appropriate favicon

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#21]
We need to have a favicon!

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by Monal5031** [comment]
Maybe a pokeball? @anshumanv

**Comment by anshumanv** [comment]
Of course!
As present in the body.

**Comment by Monal5031** [comment]
assign it please 😄

**Comment by anshumanv** [comment]
*Assigned to* **Monal5031**
Same gh bug, not able to assign **you**.

**Comment by Monal5031** [comment]
@anshumanv its not a gh bug but only contributors can be assigned AFAIK, also I think the is not effective on the webapp can you check it once?

**Comment by anshumanv** [comment]
Yes, got it, correct

**Comment by anshumanv** [comment]
It will be functional by tonight. :wink:

**Comment by anshumanv** [comment]
@Monal5031 awesome favicon! Please check. :smile:
